engine/src/main/java/org/hibernate/search/engine/spatial/DistanceUnit.java

```diff
@@ -34,4 +34,12 @@ public double toMeters(double distance) {
 
 		return distance * toMeters;
 	}
+
+	public Double metersToUnit(Double distanceInMeters) {
+		if ( distanceInMeters == null ) {
+			return null;
+		}
+
+		return distanceInMeters * 1 / toMeters;
```

Comment:
I guess there's a reason, but... why `* 1`?

Reply:
Is me having a cold a good explanation? I don't see another one :).